src/components/Navigation/SearchBar/SearchBar.vue

```diff
@@ -113,16 +114,19 @@ export default defineComponent({
       }
     };
 
-    const setActiveFilter = function(): void {
-      router.replace({
-        path: "/",
-        query: {
-          ...router.currentRoute.value.query,
-          filterPhrase: filterVal.value
-        }
-      });
-      store.commit("nav/filterFoodByCategory");
-      store.commit("nav/filterFoodList", filterVal.value);
+    const setActiveFilter = function(event: any): void {
+      if(event && event.target){
+        const filterValue = event.target.value;
```

Comment:
type

Reply:
There is later problem with typing. I have consulted this earlier with Kevin, and he advised to let it be for now